src/main/java/org/carlmontrobotics/commands/MoveToNote.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package org.carlmontrobotics.commands;
+
+import org.carlmontrobotics.Constants.Limelightc;
+import org.carlmontrobotics.subsystems.Drivetrain;
+import org.carlmontrobotics.subsystems.Limelight;
+import org.carlmontrobotics.subsystems.LimelightHelpers;
+
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class MoveToNote extends Command {
```

Comment:
This command doesn't seem to have anything to do with auto-aiming of the arm. Ideally it would be part of a separate PR.